src/pages/index.tsx

```diff
@@ -22,6 +26,7 @@ const Home: NextPage<HomeProps> = ({ title, keywords, chainId }: Readonly<HomePr
         <title>{title}</title>
         <meta content={keywords} name="keywords" />
         <link href="/okp4-logo.png" rel="icon" />
+        <meta content={publicRuntimeConfig.version} name="version" />
       </Head>
 
       <ContentWithoutSSR chainId={chainId} />
```

Comment:
I don't know if this can improve the performance, but since we already fetch`publicRuntimeConfig` here, wouldn't it be better to pass it by props to content?

Reply:
@lolottetheclash 
It's not a fetch, it's just a pointer to the `next.config.js` file, so it will no have performance issue.

Reply:
@lolottetheclash The `version` is injected at compile time. No performance issue. But, we could make it configurable with a prop though. But version for me is special, so I wouldn't make it visible outside the component. We can of course discuss about it.

Reply:
thanks for your explanations @ccamel & @fredericvilcot 👌